packages/data-service-generator/package.json

```diff
@@ -5,7 +5,7 @@
     "dsg:package:container": "./scripts/package-container.sh"
   },
   "dependencies": {
-    "@amplication/code-gen-types": "2.0.18",
+    "@amplication/code-gen-types": "2.0.19",
```

Comment:
⚠️ This PR depends on https://github.com/amplication/amplication/pull/7399 to be merged and the npm package published